src/pages/index.js

```diff
@@ -44,125 +54,220 @@ export const cardData = {
 
 export const cardListSelector = ".card";
 
-const profileEditButton = document.querySelector("#profile-edit-button");
-const profileEditModal = document.querySelector("#profile-edit-modal");
-const addCardModal = document.querySelector("#add-card-modal");
-const previewImageModal = document.querySelector("#preview-image-modal");
-const profileTitle = document.querySelector("#profile-title");
-const profileDescription = document.querySelector("#profile-description");
-// const profileEditForm = profileEditModal.querySelector(".modal__form");
-const addCardFormElement = addCardModal.querySelector(".modal__form");
-const cardListEL = document.querySelector(".card");
-// const cardTemplate =
-//   document.querySelector("#card-template").content.firstElementChild;
-const popups = document.querySelectorAll(".modal");
-
-/* Buttons Elements */
-const profileCloseButton = document.querySelector("#profile-close-button");
-const cardCloseButton = document.querySelector("#card-close-button");
-const addNewCardButton = document.querySelector(".profile__add-button");
-// const closePreviewButton = document.querySelector(
-//   "#preview-close-image-button"
-// );
-
-/* Form Data const*/
-const userInfo = new UserInfo(profileTitle, profileDescription);
-
-const profileDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-const profileTitleInput = document.querySelector("#profile-title-input");
-const cardTitleInput = addCardFormElement.querySelector(
-  ".modal__input_type_title"
+Promise.all([api.getUserInfo(), api.getInitialCards()]).then(
+  ([userData, cardData]) => {
+    userInfo.setUserInfo({ name: userData.name, description: userData.about });
+    userInfo.setUserAvatar(userData.avatar);
+  }
 );
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**